data/commands/help.json

```diff
@@ -32,14 +33,21 @@
     ],
 
     "Earning Money": [
-      "- Invest in properties - see /properties",
+      "- Invest in properties - see /properties", 
       "- Export vehicles - see /export",
       "- Win minigames - see /minigames",
       "- Complete delivery missions - see /deliver",
       "- Crush vehicles at the scrap yard, north of the LV Stadium",
       "- Find the Bag of Cash",
       "- Take out a loan - see /borrow"
     ],
+   "Gang Stuff": [
+   "Usage: /gang [create/invite/join/kick/leave/members/settings/transactions",
+   "Usage: /gbank, /gwithdraw, /gbalance",
+   "",
+   "Wants to create a gang zone?",
```

Comment:
Not to be a pitch-perfect guy. But we don't want others to make fun of our grammar mistakes. 
so it should be `"Want to create a gang zone?",`.  Other than that. your initiative is great. Let's see what others have to approve of this.